gstd/gstd.c

```diff
@@ -100,6 +101,10 @@ main (gint argc, gchar * argv[])
     {"gst-log-filename", 'd', 0, G_OPTION_ARG_FILENAME, &gstlogfile,
         "Create gst.log file to path", NULL}
     ,
+    {"no-log", 'L', 0, G_OPTION_ARG_NONE, &nolog,
```

Comment:
`-L` for the option of disabling the logging is counter-intuitive. Should we add the boolean as an argument and make it enabled by default?